packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -484,4 +484,44 @@
       <label>Voir moins</label>
       <name>quantic_SmartSnippetShowLess</name>
     </customLabels>
+    <customLabels>
+      <label>Expliquer pourquoi</label>
```

Comment:
I don't want to nitpick but I was wondering for consistency do we want "Expliquez" instead? I see you are already using 'Sélectionnez' or maybe the latter should end with 'er'?

Reply:
nitpicking is what PR reviews are for 👌 thanks!
will ask @acarpentier01  for help to make sure the labels are correct!